lib/V8/v8-utils.cpp

```diff
@@ -202,6 +204,85 @@ static void CreateErrorObject(v8::Isolate* isolate, int errorNumber,
   }
 }
 
+////////////////////////////////////////////////////////////////////////////////
+/// @brief set a point in time after which we will abort external connection
+////////////////////////////////////////////////////////////////////////////////
+static std::atomic<double> connectionToBeDeadAt(0.0);
```

Comment:
`dead` is a wrong adjective here, as the connection is not dead.
The variable name should rather be `connectionExpirationStamp` or something like this.

Reply:
Additionally, as all of this seems to be used from within the arangosh only, can we move the code from `lib` into the `arangosh` folder?

Reply:
the simple http client is used on both sides :/